Esri/arcgis-maps-sdk-swift-samples
# [Release] 200.0.0-beta

## Description

This PR releases the sample viewer app with `200.0.0-beta` version of [ArcGIS Maps SDK for Swift](https://developers.arcgis.com/swift/). A few more things todo:

- [ ] Copy-paste release notes and tag a draft release
- [ ] Update the sample viewer using GitHub Swift package, instead of local Swift package
- [ ] Verify the version strings in About page and project settings
- [ ] Proofread root README instructions
- [ ] Hit the green button and make draft official 🎉 

## Linked Issue(s)

- `common-samples/issues/3804`

## How To Test

Before hitting the green button, make sure at least 2 more people tested the build passes on their various devices.
